playground/switch-test.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Good stuff splitting out the tests! One nit, The src folder is looking a bit full. We might want to separate the .ts .vue and .html files into folders for better organization at some point.

Reply:
will fix this in the next e2e test PR!